src/Build/Resources/Strings.resx

```diff
@@ -463,6 +463,10 @@
     likely because of a programming error in the logger). When a logger dies, we cannot proceed with the build, and we throw a
     special exception to abort the build.</comment>
   </data>
+  <data name="LoggingBeforeTaskInitialization" UESanitized="false" Visibility="Public">
```

Comment:
this is thrown out of the MSBuild intrinsic task, which is in the engine side. copy/pasted the resource from utilities over.